src/components/Dashboard/FilingHistoryList/FilingTemplate.vue

```diff
@@ -53,7 +53,7 @@
       </div>
     </v-expansion-panel-header>
 
-    <v-expansion-panel-content class="pt-5">
+    <v-expansion-panel-content eager class="pt-5">
```

Comment:
Based on discussion thread below added `eager` to the `v-expansion-panel-content`. This way panel content is generated.
https://github.com/vuejs/vue-test-utils/issues/1827